src/sonic-yang-models/yang-templates/sonic-types.yang.j2

```diff
@@ -66,6 +66,7 @@ module sonic-types {
             enum REDIRECT;
             enum DO_NOT_NAT;
             enum DISABLE_TRIM;
+            enum COPY;
```

Comment:
@devpatha Could you add yang test for COPY ACL action

Reply:
@kperumalbfn I added tests for COPY ACL action. Could you please review it?